### PR TITLE
Upgrade Literate and Documenter.

### PR DESCRIPTION
**Pull request by fredrikekre**
Testing for https://github.com/fredrikekre/Literate.jl/pull/84

### CODE REVIEW

**Comment by maleadt** [comment]
bors try

**Comment by maleadt** [comment]
Let's just try again?
bors try

**Comment by fredrikekre** [comment]
I repurposed this PR; now updates Literate and Documenter which should fix the broken links.

File: docs/Manifest.toml [hunk 1]
```diff
@@ -51,9 +51,9 @@ uuid = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 
 [[Literate]]
 deps = ["Base64", "JSON", "REPL"]
-git-tree-sha1 = "707c58359f2de555ace074313baea957c3187f2b"
+path = "/home/fredrik/dev/Literate"
```
Comment (fredrikekre) [inline docs/Manifest.toml]: :slightly_smiling_face:

**Comment by maleadt** [comment]
Thanks for looking into this!
bors r+

**Comment by fredrikekre** [comment]
Compat@3 strikes again.

**Comment by maleadt** [comment]
master branch Manifest was broken because of it, works again after a `Pkg.up`. Rebase and try again?

**Comment by fredrikekre** [comment]
Done.

**Comment by maleadt** [comment]
I disabled the broken test, no idea why it fails...
bors try

**Pull request merged**